### PR TITLE
Welcome Command

### PR DESCRIPTION
**Pull request by krogers78**
This feature sends a direct message to the user welcoming them to the server and telling them the different methods of earning points. If the user is not verified they will see a message saying "Get verified to start earning points" while if a user issues the welcome command and is verified they will see "Start earning points."

### CODE REVIEW

File: bot/botUtils.js [hunk 1]
```diff
@@ -33,10 +34,47 @@ exports.isAdmin = (member) => {
...
+  util.log('User id', user.id, 4);
+  let welcomeString = null;
+  const pointsMethods = `
+    **How to Earn Points:**
+    \n\`Chatting with friends\`
+    Chat with others and earn points! Get rewarded for talking to others!
+    \n\`Gaming, Streaming, and More!\`
+    Be rewarded for engagement through streaming, playing games with others, listening to Spotify, and more!
+    \n\`Invite New Members\`
+    Invite new members to the group and be rewarded with points!
```
Comment (BHess2653) [inline bot/botUtils.js]: should there be anything here about only full sail students?

Comment (krogers78) [reply bot/botUtils.js]: You're right there should be. I will change it to include something about Full Sail Students.

Comment (BHess2653) [reply bot/botUtils.js]: other than that it looks good to me

File: bot/botUtils.js [hunk 2]
```diff
@@ -33,10 +34,47 @@ exports.isAdmin = (member) => {
...
 // Update the user points in the database
 exports.getUserPointsandUpdate = async (userId, pointsToAdd) => {
-  const memberData = await Member.findAll({ attributes: ['points'], where: { discordUser: userId } });
-  await Member.update(
-    { points: memberData[0].dataValues.points + pointsToAdd },
-    { where: { discordUser: userId } });
+  const memberData = await Member.findAll({ attributes: ['points', 'verified'], where: { discordUser: userId } });
+  const { points, verified } = memberData[0].dataValues;
+  const pointsBeingAdded = parseFloat((points + pointsToAdd).toFixed(2));
+  verified ? await Member.update(
```
Comment (MikeSewell) [inline bot/botUtils.js]: I'm not sure what this line is doing.

Comment (krogers78) [reply bot/botUtils.js]: This line checks if they're verified in the database in order to only award points if they're verified. Do you think it needs to be changed to be more clear?

Comment (MikeSewell) [reply bot/botUtils.js]: Don't change it, but put a comment on it so that others can learn this type of conditional.

**Review by MikeSewell — APPROVED** [review]

**Pull request merged**